<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionHandler</class>
 <widget class="QDialog" name="SessionHandler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="SessionName">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 17pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TeacherAttendanceTableWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>931</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="GroupComboBox">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>120</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Group</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="GroupLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
  <widget class="QLabel" name="TeacherLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Teacher</string>
   </property>
  </widget>
  <widget class="QLabel" name="TeacherAttendance">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Teachers Attendance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TeacherlineEdit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>361</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="TeacherListWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UnassignButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resources for SMS/unassign_resized.jpeg);
background-position: center;
background-repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
